airflow/operators/bash.py

```diff
@@ -72,6 +72,10 @@ class BashOperator(BaseOperator):
         users in the Web UI. Most of the default template variables are not at
         risk.
 
+        Add a space after the script name when directly calling a Bash script with the
+        ``bash_command`` argument.  This is because Airflow tries to apply a Jinja template to it,
+        which will fail.
+
```

Comment:
You've placed this in the middle of a warning, it needs to be moved else where, perhaps above the `.. wanring::` and in to a `.. note::` block